cairo/ethereum/cancun/vm/precompiled_contracts/alt_bn128.cairo

```diff
@@ -161,3 +233,24 @@ func alt_bn128_pairing_check{
     tempvar ok = cast(0, EthereumException*);
     return ok;
 }
+
+func alt_bn128_G1Point__to_Bytes_be{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
```

Comment:
is this related to altbn128 or G1Point generally speaking?

Reply:
altbn128's G1 point as we know it's expressed over 32 bytes (which might not be valid for other curves)